templates/appliance/community.html

```diff
@@ -1,216 +1,213 @@
 {% extends "appliance/_base-appliance.html" %}
 
+{% from "_macros/vf_hero.jinja" import vf_hero %}
+{% from "_macros/vf_tiered-list.jinja" import vf_tiered_list %}
+
 {% block title %}Ubuntu Appliance Community - Governance and Policies{% endblock %}
```

Comment:
According to the doc, this is the correct page title, but I'd double check with @Sophie-32 or @eliman11 is the original sounds way more descriptive 
```suggestion
{% block title %}Community{% endblock %}
```